.vscode/extensions.json

```diff
@@ -1,5 +1,7 @@
 {
     "recommendations": [
-        "ms-vscode.csharp"
+        "ms-vscode.csharp",
+        "DavidAnson.vscode-markdownlint",
+        "ms-vscode.csharp",
```

Comment:
usually we don't include the .vscode directory in git

Reply:
This file provides suggestions of the extensions to download. I think it is useful, specially for new users.

Reply:
fine with me :)